pkgs/development/python-modules/camxes/default.nix

```diff
@@ -0,0 +1,53 @@
+{ lib
+, jre
+, buildPythonPackage
+, fetchPypi
+, progressbar
+}:
+
+let
+  LEPL = buildPythonPackage rec {
```

Comment:
No "private" packages in the Python packages set. This should be called from `python-packages.nix`.